engine/paymail/paymail_payload_format.go

```diff
@@ -0,0 +1,25 @@
+package paymail
+
+import "fmt"
+
+// PayloadFormat is the format of the paymail payload
+type PayloadFormat uint32
```

Comment:
Are there any requirements that you've decided to use `uint32`?
Standard `int` would be "more normal xD"

Reply:
It is done this way in engine, and I don't see good reason to change this.

Reply:
Man, How I miss the real enums in Go